### DISCUSSION TITLE
spongy mouse... (maybe no raw input?)

### DISCUSSION THREAD

**Discussion by Blazzycrafter** [ppy/osu#20722] [Q&A] [answered]
### Type

Game behaviour

### Bug description

my mouse is "spongy"
i dont know what is it...
but its nor raw input

### Screenshots or videos

too big....
ytlink:
https://youtu.be/3Fmtvba468Q

Prof its the same problem with "raw input" (high presistion mouse)
https://youtu.be/xM7n_AHQJKU

### Version

2022.1008.2-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9762421/database.log)
[updater.log](https://github.com/ppy/osu/files/9762422/updater.log)
[runtime.log](https://github.com/ppy/osu/files/9762423/runtime.log)
[network.log](https://github.com/ppy/osu/files/9762424/network.log)
[performance.log](https://github.com/ppy/osu/files/9762425/performance.log)

**Comment by Blazzycrafter** [comment]
i dont know why but this ISSUE
was instantly moved to a discustion... in type Q&A
thanks peppy -_- or your autonmatic bot...
![image](https://user-images.githubusercontent.com/39300111/195275874-a3151655-0fd1-4009-b242-550c6c75b0c7.png)

**Reply by peppy** [reply]
Yep, we keep things in discussions when they aren't bugs. Thanks for understanding.

**Comment by peppy** [comment, marked as answer]
The moment you alt-tab, the game will drop its frame rate intentionally to conserve resources. The testing methodology you are using in the video is therefore flawed.

But in addition, it seems like you aren't running exclusive fullscreen. There will always be several frames of latency induced if you don't run exclusive fullscreen due to the window manager buffer, so please change your setting to fix the issue.

**Reply by Blazzycrafter** [reply]
> The moment you alt-tab, the game will drop its frame rate intentionally to conserve resources. The testing methodology you are using in the video is therefore flawed.

yes but i had seen that osu classic also decreases the fps with alt+tab
so i saw a possibilityo test it there...
 
> But in addition, it seems like you aren't running exclusive fullscreen. There will always be several frames of latency induced if you don't run exclusive fullscreen due to the window manager buffer, so please change your setting to fix the issue.

ok yeah with fullscreen its works... right...
but why the heck does lazer have a noticeably higher input latency in the borderless fenter than on classic?

**Reply by peppy** [reply]
> but why the heck does lazer have a noticeably higher input latency in the borderless fenter than on classic?

Maybe you're running stable in a mode (like compatibility mode) which automatically uses exclusive fullscreen even when you select borderless. That's a feature of Direct X.

And if you're testing while alt-tabbed as a benchmark... I don't know what to say. The two apps are probably doing things differently. Stable might still be running at full rate. Who knows.

**Reply by Blazzycrafter** [reply]
¯\\_(ツ)_/¯

**Comment by ToxicMushroom** [comment]
I've the same issue on linux, osu-stable through wine feels a lot more responsive, I can run both windowed or in fullscreen and stable just feels great either way, lazer does not it.
I think spongy is a good way to describe how it feels.

**Reply by mhdimo** [reply]
more than spongy id say it feels sluggish... and even if running with fullscreen or not it doesn't really change the results it still has the same feel

**Reply by peppy** [reply]
On linux, I have no idea how your framebuffer is setup, but it can vary depending on your window server and probably more variables. I'm guessing it will also affect other games in a similar way. Hard for us to support linux because of the number of variables, but do feel free to try and figure out the correct configuration for your system to make it work with applications.

**Reply by smoogipoo** [reply]
If anything on Linux it's going to be your compositor, and whether it's running through X11 or Wayland. But yes it's something we can't really support and is on a per-system/per-os/per-configuration basis.
